xrootd.spec

```diff
@@ -8,7 +8,7 @@
 %define github_user xrootd
 Source: git+https://github.com/%github_user/xrootd.git?obj=%{branch}/v%{tag}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}.tgz
```

Comment:
Please use `Source: https://xrootd.slac.stanford.edu/download/v%{version}/%{name}-%{version}.tar.gz`, which corresponds to the official tarball. I think you may have problems using a dynamically generated tarball from git.